pkg/sql/logic_test.go

```diff
@@ -454,6 +454,12 @@ type logicQuery struct {
 	// expectedErr and expectedErrCode are as in logicStatement.
 	expectErr     string
 	expectErrCode string
+
+	// if set, the results are cross-checked against previous queries with the
+	// same label.
+	label string
```

Comment:
Maybe I'm just blind, but where is this set? Also, is this change used for the next commit?